packages/simcore-sdk/tests/integration/test_node_ports_common_filemanager.py

```diff
@@ -291,14 +302,16 @@ async def test_invalid_file_path(
 
     download_folder = Path(tmpdir) / "downloads"
     with pytest.raises(exceptions.S3InvalidPathError):
-        await filemanager.download_file_from_s3(
-            user_id=user_id,
-            store_id=store,
-            store_name=None,
-            s3_object=file_id,
-            local_folder=download_folder,
-            io_log_redirect_cb=None,
-        )
+        async with ProgressBarData(steps=1) as progress_bar:
+            await filemanager.download_file_from_s3(
+                user_id=user_id,
+                store_id=store,
+                store_name=None,
+                s3_object=file_id,
+                local_folder=download_folder,
+                io_log_redirect_cb=None,
+                progress_bar=progress_bar,

```

Comment:
Question: Why is the progress bar required here where it raises an error?

Reply:
hmm because it is part of the function signature?